### PR TITLE
A few small fixes

### PR DESCRIPTION
**Pull request by ismailmustafa**
- Update numbering in workflows section
- Fix passkey type in JS SDK reference

### CODE REVIEW

File: docs/workflows/account-setup.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Account Setup
-sidebar_position: 2
+sidebar_position: 0
```
Comment (turnipdabeets) [inline docs/workflows/account-setup.md:3]: Are we still planning to use an intro workflow from Colton?

File: docs/workflows/api-token.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: API Tokens
-sidebar_position: 6
+sidebar_position: 4
```
Comment (turnipdabeets) [inline docs/workflows/api-token.md:3]: The Realms and Application guides link to this tokens guide in the creating them by API sections. Probably fine to link this guide after them though.

**Review by turnipdabeets — APPROVED** [review]

File: docs/workflows/account-setup.md (hunk 1 above)
Comment (ismailmustafa) [reply docs/workflows/account-setup.md:3]: No, we're not going to have an intro anymore

File: docs/workflows/api-token.md (hunk 2 above)
Comment (ismailmustafa) [reply docs/workflows/api-token.md:3]: How does that impact numbering?

Comment (turnipdabeets) [reply docs/workflows/api-token.md:3]: The API Tokens guide is displayed after the Realms and Application guides based on numbering

Comment (ismailmustafa) [reply docs/workflows/api-token.md:3]: Oh I see. What do you think a good ordering would be?

Comment (turnipdabeets) [reply docs/workflows/api-token.md:3]: I'm not sure. I think what you have is probably fine. I was just pointing out that the API Token workflow would be listed after 2 guides that link to with this new order, but it's probably fine to do that.

Comment (ismailmustafa) [reply docs/workflows/api-token.md:3]: The ordering here is identical to what it currently is in production, except for the bind and authenticate guides. Do you think we should move API tokens to after account setup?

Comment (turnipdabeets) [reply docs/workflows/api-token.md:3]: I think it's fine either way. I'll resolve this so you can merge.

**Pull request merged**